README.md

```diff
@@ -1,2 +1,17 @@
 # cheri-instruction-rcg-gen
-Generate Runtime Code Generation library from llvm-tblgen json instruction spec
+Generate Runtime Code Generation library from json descriptions extracted from llvm-tblgen using `--dump-json`
+
+
+Build:
+
+```bash
```

Comment:
Can we make these just `sh` (they're not really bash specific, and some of us don't even use bash :) ).

Reply:
done